R/run.R

```diff
@@ -266,8 +269,10 @@ orderly_run <- function(name, parameters = NULL, envir = NULL, echo = TRUE,
 
 custom_metadata <- function(dat) {
   shared <- dat$shared_resources %||% list()
+  file_names <- get_orderly_file_names(dat$src)
+  file_name <- file_names[file_exists(file_names, workdir = dat$src)]
```

Comment:
I wonder if worth getting this into the `orderly2` metadata on the packet. See line 208 - 210 in `run.R` the filename is available to you there, put it into that list and then pull it out here.